tutorials/scripting/gdscript/warnings/get_node_default_without_onready.rst

```diff
@@ -0,0 +1,63 @@
+GET_NODE_DEFAULT_WITHOUT_ONREADY
+====================================
+
+This warning appears when the default value of a node's property is set to a location in the scene tree without using the ``@onready`` annotation.
```

Comment:
This reads as... well, really overly technical to me. It took me a couple re-reads to figure out what it was trying to tell me.

Maybe:

```suggestion
This warning appears when a node's property is initialized to a location in the scene tree without using the ``@onready`` annotation.
```

Reply:
I like the suggestion for the sentence and will add that in, thanks!

As for the "property" vs "member variable" situation, I think I was mostly inspired by the class reference, where they are referred to as "properties". Granted, those are for built-in classes that are more of a black box for users, whereas the things we're talking about here users have direct access to modify and inspect. But for now, "property" is more succinct and used frequently in documentation, so I think it makes sense to keep it that way.